### PR TITLE
(#43) allow user get all cookies

### PR DESCRIPTION
**Pull request by dannypsnl**
API: `func (*Cookies) List() []*http.Cookie`

### CODE REVIEW

File: cookies.go [hunk 1]
```diff
@@ -11,3 +11,7 @@ type Cookies struct {
 func (c *Cookies) Cookie(name string) (*http.Cookie, error) {
```
Comment (shana0440) [inline cookies.go]: maybe `Get` will better than `Cookie`?

Comment (dannypsnl) [reply cookies.go]: In fact, I prefer `cookie["name"]`, would it be too tricky?

Comment (shana0440) [reply cookies.go]: `cookie["name"]` like it can set value.
I think it was a little tricky.

Comment (dannypsnl) [reply cookies.go]: changed at: 130910c

**Comment by dannypsnl** [comment]
@shana0440 if looks ok to you, please merge & delete the branch

**Pull request merged**